src/gccrs/mod.rs

```diff
@@ -75,7 +75,11 @@ impl Gccrs {
         Gccrs::fake_output(r#"lib___.so"#);
 
         Gccrs::dump_config()?;
-        GccrsConfig::display()
+        let config = GccrsConfig::new()?;
+
+        println!("{}", config);
```

Comment:
does this mean we print the config each time we invoke cargo-gccrs?

Reply:
Also is there a logging interface we should be using for cargo? I don't think you should change anything if there is, just out of curiosity on my part.

Reply:
> does this mean we print the config each time we invoke cargo-gccrs?

The config printing is only done when `cargo` initially expects the compiler to dump its configuration. So this only happens once, when initially compiling a project. You can see that `cfg_print()` is only called when `cargo` orders us to compile from stdin, which can only happen in that case. I am gonna refactor this to use the rustc option parser later on.

> Also is there a logging interface we should be using for cargo? I don't think you should change anything if there is, just out of curiosity on my part.

No idea about this, sorry! Maybe @flip1995 or @bjorn3 might know

Reply:
> Also is there a logging interface we should be using for cargo?

I don't think there is. But I'm far from a cargo expert.

Reply:
I don't think there is either. I am not a cargo expert either though.